src/App.test.jsx

```diff
@@ -0,0 +1,20 @@
+import App from './App';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+test('When button is pressed a new item is added', () => {
+  render(<App />);
+
+  const itemInput = screen.getByRole('textbox');
```

Comment:
These getBys are assertions themselves, so it's up to you re: readability if you want to add a line for the toBeInTheDocument()